docs/Community/SecurityAdvisoryHandling.md

```diff
@@ -0,0 +1,83 @@
+   1. The assignee(s) responsible for fixing the issue.
+   2. Any area owners or subject matter experts needed for review.
+   3. CI/CD coordinators for private testing and integration with the release. (e.g., [@brmclaren](https://github.com/brmclaren)).
+4. [Create a temporary private fork](https://docs.github.com/en/github/managing-security-vulnerabilities/collaborating-in-a-temporary-private-fork-to-resolve-a-security-vulnerability#creating-a-temporary-private-fork)
+   to implement the code changes as part of the SA.
+   1. The assignee should follow the GitHub instructions and create an `advisory-fix` branch for their code changes.
+      1. The code changes should include an update to [CHANGELOG.md](/CHANGELOG.md) under the `### Security` subheader describing the issue addressed.
+   2. Once the changes are ready for review, they should be submitted as a PR to the master branch of the temporary fork.
+      1. GitHub does not currently appear to support requiring PR reviews to merge into the master branch of a temporary fork, so maintainers will need to enforce this process manually.
+5. Reach out to the CI/CD maintainers to establish a private CI/CD pipeline for testing code changes.
+
+By default, all SAs are targeted for inclusion into the next release of the OE SDK, which usually happens quarterly.
+If the fix cannot be implemented or verified in time for an upcoming release, it can be included in a following patch version release.
+The security reviewers are responsible for deciding if a patch release is needed to address a vulnerability before its scheduled public disclosure date.
+
+Once the security fix is reviewed, tested, and merged into the temporary fork master branch for release,
+the assignee and the release manager will coordinate integrating the fix into the release branch.
+This usually includes:
+
+1. Rebasing the temporary fork master branch to the project's master branch and testing the merged branch in private CI/CD.
+2. Publishing the SA and merging its master branch into the project master branch.
+   1. At this point, the vulnerability is considered publicly disclosed, and a new version of the OE SDK packages with the fix should be made available as quickly as possible.
+3. Cherry-picking the fix to the release branch.
+4. Building and releasing the OE SDK packages with the fix.
+5. Updating the published SA with a pointer to the released packages containing the fix.
+
+## Disclosure
+
+The Open Enclave SDK project currently relies on the principles for [Coordinated Vulnerability Disclosure (CVD)](https://www.microsoft.com/en-us/msrc/cvd) put forth by MSRC.
+The CVD includes a couple of policies such as:
+
+- acknowledgment of vulnerability reports within 24 hours (via MSRC)
+- supporting public disclosure within 90 days as the baseline, and negotiating that with vulnerability reporters and necessary
```

Comment:
incomplete sentence here. necessary "trusted stakeholders"?

Reply:
Not sure how this line got truncated; it's supposed to say "necessary parties" which would include engineering teams and vendors as discussed in the CVD. 

The "trusted stakeholder" term is only introduced after this to more specifically discuss a group OE as an open source project needs to define for its governance.